de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/HTMLWhiteboardView.java

```diff
@@ -1,18 +1,28 @@
 package de.fu_berlin.inf.dpp.whiteboard.ui;
 
+import java.io.IOException;
+import java.net.URL;
+
 import org.apache.log4j.Logger;
+import org.eclipse.core.runtime.FileLocator;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.browser.Browser;
+import org.eclipse.swt.browser.ProgressEvent;
+import org.eclipse.swt.browser.ProgressListener;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.ui.part.ViewPart;
 
+import de.fu_berlin.inf.dpp.whiteboard.net.WhiteboardManager;
+import de.fu_berlin.inf.dpp.whiteboard.ui.browser.EclipseWhiteboardBrowser;
+
 /**
  * This is the main view for the HTML Whiteboard, it contains the browser which
  * runs the HTML Whiteboard code
  * 
  */
 public class HTMLWhiteboardView extends ViewPart {
 
+    private static final String PATH_TO_HTML_WHITEBOARD = "/frontend/dist/index.html";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=296&issues=AWbew9FqO08i52vdMn35&open=AWbew9FqO08i52vdMn35)

[ISSUE_KEY]: AWbew9FqO08i52vdMn35